examples/customresources/demos/complexmodule/module.go

```diff
@@ -15,10 +15,8 @@ import (
 )
 
 func main() {
-	// We will use a logging.Logger named "complexmodule," and our module will support 4 different
-	// resource models.
+	// ModularMain will stand up a module which will host 4 different models.
 	module.ModularMain(
-		"complexmodule",
```

Comment:
sighhhhhhhhh

Reply:
@cheukt  I'm fine with leaving `ModularMain` alone. And just leaving a small comment about why it takes in a string that isn't used.

Reply:
no it's okay, I'm being slightly melodramatic here. We will update the go modules eventually. 

Will anything break the binaries currently uploaded to the registry? from my understanding, no.

Reply:
no, it will not break any current binaries, this will only break if you update RDK on the package